src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -414,8 +414,8 @@ void ObjectMgr::LoadCreatureTemplates()
                                              "spell2, spell3, spell4, spell5, spell6, spell7, spell8, PetSpellDataId, VehicleId, mingold, maxgold, AIName, MovementType, "
     //                                        62           63           64              65            66             67              68                  69
                                              "InhabitType, HoverHeight, HealthModifier, ManaModifier, ArmorModifier, DamageModifier, ExperienceModifier, RacialLeader, "
-    //                                             70          71                72               73          74
-                                             "movementId, RegenHealth, mechanic_immune_mask, flags_extra, ScriptName "
+    //                                             70          71                72                    73                  74           75
```

Comment:
I know this is cosmetic (comment), but if you look at the numbering before 70, you will see that the numbers are placed at the same column as the first letter in the field name. Would you mind aligning the numbers to the previous (untouched) lines, even if this is just a comment line?

Reply:
Sure! Didn't notice it at first. Thanks.